Archisinal.md

```diff
@@ -0,0 +1,310 @@
+
+*Marketing:* We plan to market DeCon v2 to architecture and engineering practices through targeted advertising, content marketing, and direct outreach. We will also attend industry events and conferences to increase our visibility and generate leads.
+
+## DeCon v3
+
+DeCon v3 involves construction yield farming and covers project overspend through yield farming and liquid staking of escrowed project funds. This stage of the product development involves the following key areas: technical development, business strategy, and fundraising.
+
+*Technical Development:* We plan to develop the construction yield farming platform using substrate. The platform will enable users to stake cryptocurrency and earn rewards in exchange for supporting construction projects. Additionally, we plan to integrate a liquid staking feature into the platform, which will enable users to access their staked funds quickly and easily.
+
+*Business Strategy:* Our business strategy for DeCon v3 is to offer the construction yield farming platform as a value-added service for architecture and engineering practices. We plan to generate revenue by charging a small percentage of the rewards earned by users on the platform.
+
+*Fundraising:* We plan to raise funds for DeCon v3 through a series B round of funding. We will pitch to venture capital firms, angel investors, and other investors who are interested in blockchain technology and the construction industry.
+
+*Marketing:* We plan to market DeCon v3 to construction companies, architects, and engineering firms through targeted advertising, content marketing, and direct outreach. We will also attend industry events and conferences to increase our visibility and generate leads.
+
+## Conclusion
+
+In conclusion, the Archisinal roadmap outlines the development of a fully functioning NFT marketplace for architects and designers to mint and trade their designs, and the pivot to a B2B solution that provides document control and timestamping services for architecture and engineering practices in commercial and residential construction projects. We will introduce the ArchisinalDAO platform and the dispute resolution feature, develop a payment platform and a whole project management system, and create a construction yield farming platform. Our business strategy involves generating revenue through subscription-based services and partnerships with construction companies, architects, and engineering firms. We will also raise funds through seed, series A, and series B rounds of funding, and market our product through targeted advertising, content marketing, and direct outreach.
+
+## Additional Information :heavy_plus_sign:
+
+**How did you hear about the Bounty Program?** Announcement by another team
+
+- The Astar Grants program was highlighted through a Medium blogpost.
+- Basic POC entry in the Encode Club Spring 2023 Hackathon - [Here](https://github.com/Archisinal/Encode_Hackathon)
```

Comment:
This link 404s

Reply:
Thanks for changing the link. Can you also post a video to Loom or something showing the functionality? It makes it easier (as opposed to me having to connect my wallet, get testnet tokens, etc.)

Reply:
[Screencast from 05-08-2023 03_48_40 PM (3).webm](https://github.com/use-inkubator/Ecosystem-Grants/assets/123243302/426364eb-8abf-4aeb-ab8b-f377aed87d6a)